pkg/virt-operator/util/config.go

```diff
@@ -24,20 +24,32 @@ import (
 	"regexp"
 	"strings"
 
+	k8sv1 "k8s.io/api/core/v1"
+
 	kvutil "kubevirt.io/kubevirt/pkg/util"
 )
 
 const (
 	// Name of env var containing the operator's image name
 	OperatorImageEnvName   = "OPERATOR_IMAGE"
 	TargetInstallNamespace = "TARGET_INSTALL_NAMESPACE"
+	TargetImagePullPolicy  = "TARGET_IMAGE_PULL_POLICY"
 )
 
 type KubeVirtDeploymentConfig struct {
 	ImageRegistry string
 	ImageTag      string
 }
 
+func GetTargetImagePullPolicy() k8sv1.PullPolicy {
+	pullPolicy := os.Getenv(TargetImagePullPolicy)
```

Comment:
I didn't follow this too  much, so therefore my question: Should this be set as an environment variable, or like the PR title suggests, as a filed on the CR?

Reply:
the option is set on the CR.

That option results in a env var being passed to the job that creates the install strategy

Reply:
remember, we're using the virt-operator container as a k8s job to generate/dump an install strategy to a configMap